AY2425S1-CS2103T-W09-3/tp
# Fix alpha bugs and improve UI

## Description

Closes #182, #149, #144





## Changes made





Added filter to summary in UG, specify that filter does not stack with find. 
Added HireMeTheme.css, changed stylesheet of MainWindow.fxml
## Impact of changes

Changed color of hireme application

## Testing Steps

1. Check the HiremeTheme.css in resources/view
2. Run the app and help check if there are bugs
